packages/default/scss/chip/_layout.scss

```diff
@@ -90,7 +91,15 @@
     }
 
     .k-chip-content {
+        min-width: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
         padding: 0 $chip-content-padding-x;
+        flex: 1;
+    }
+
+    .k-chip-label {
+        white-space: nowrap;
```

Comment:
I think it will be better if we move the `white-space: nowrap` to `.k-chip-content` because we do not have `.k-chip-label` element when the user adds the label with content projection.
An example of what I mean can be seen in the [Custom Content](https://www.telerik.com/kendo-angular-ui/components/buttons/chip/content/#toc-custom-content) demo.

Reply:
Good catch! Thank you :)